scripts/index.js

```diff
@@ -23,6 +23,7 @@ const serveCommand = argv => {
     const url = chalk.underline(`http://${host}:${port}`);
     const message = chalk.keyword('green').bold(`Webdash running on ${url}!`);
     console.log(message);
+    opn(url)
```

Comment:
okay here's why it's not working:
url is formatted text from `chalk.underline`. So you want to save the `url` before formatting it, and then after formatting it
and `opn` should only take `url` (unformatted)

also please wrap `opn(url)` with a try catch:

```javascript
try {
opn(url);
} catch(error){}
```
because if it didn't work, it's not a big deal, and users might think that webdash is broken if `opn` broke (whereas webdash is still working)